hugegraph-pd/hg-pd-test/src/main/java/org/apache/hugegraph/pd/StoreNodeServiceTest.java

```diff
@@ -420,7 +425,7 @@ public void onPartitionRemoved(Metapb.Partition partition) {
 
     }
 
-    // @Test
```

Comment:
if a test is hard to ensure the meaning or currently we can't run it well, we could add `@Ignore` for it rather than comment `@Test`  (so
 that we could search ignore in future)

maybe we could also add a annotation `@useless` for other useless classes/method?